### PR TITLE
Add chrome.cookies.getAll to suppress Honey errors

### PR DESCRIPTION
**Pull request by kevinlawler**

### CODE REVIEW

**Comment by jonathansampson** [comment]
Appears to work well for me; @bridiver, any reason not to merge? :)

**Comment by jonathansampson** [comment]
@kevinlawler This could use a quick rebase.

**Comment by kevinlawler** [comment]
rebased/merged

File: atom/common/api/resources/cookies_bindings.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var binding = require('binding').Binding.create('cookies')
+
+binding.registerCustomHook(function (bindingsAPI, extensionId) {
+  // var apiFunctions = bindingsAPI.apiFunctions
+  var cookies = bindingsAPI.compiledApi
+
+  cookies.getAll = function (details, cb) {
```
Comment (bridiver) [inline atom/common/api/resources/cookies_bindings.js]: this should use `setHandleRequest` - see tab_bindings.js

**Comment by bridiver** [comment]
can you rebase this on 2.58.9?

**Comment by darkdh** [comment]
2.58.9 already contains the lint fix in this PR.

**Comment by kevinlawler** [comment]
will get both fixes

**Pull request merged**